src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -219,6 +228,76 @@ class HMPPSDomainEventsEmitter(
       .withAdditionalInformation("subType", event.caseNoteSubType)
   }
 
+  private fun toNonAssociationDetails(event: NonAssociationDetailsOffenderEvent): HmppsDomainEvent = HmppsDomainEvent(
+    eventType = "prison-offender-events.prisoner.non-association-detail.changed",
+    description = "A prisoner non-association detail record has changed",
+    occurredAt = event.toOccurredAt(),
+    publishedAt = OffsetDateTime.now().toString(),
+    personReference = PersonReference(event.offenderIdDisplay ?: ""),
+  )
+    .withAdditionalInformation("nomsNumber", event.offenderIdDisplay)
+    .withAdditionalInformation("bookingId", event.bookingId)
```

Comment:
I decided to publish all available info (even Nomis-specific ids) for now. Can get client feedback on whether that is useful